src/Core/Commander/Providers/WMS_Provider.js

```diff
@@ -73,6 +58,7 @@ WMS_Provider.prototype.preprocessDataLayer = function preprocessDataLayer(layer)
     layer.style = layer.style || '';
     layer.transparent = layer.transparent || false;
     layer.bbox = layer.bbox || new BoundingBox();
+    layer.options = { tileMatrixSet: 'WGS84G' };
```

Comment:
But: 
- they might not be tiled like this. For instance the Lyon opendata wms server is in meter and you can query any bbox inside its extents so the `tileMatrixSet` makes no sense in this case.
- if you add an option use the `layer.foo = layer.foo || my_default_option`